src/components/PoolInfoCard.tsx

```diff
@@ -204,6 +209,27 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
             </Box>
           ))}
         </Box>
+        <Typography mt={3}>
+          Pool address:&nbsp;
+          <Link
+            href={`https://etherscan.io/address/${poolAddress}`}
+            target="_blank"
+            color="inherit"
+          >
+            {shortenAddress(poolAddress)}
+          </Link>
+        </Typography>
+        <Typography mt={1}>
+          LP token address:&nbsp;
+          <Link
+            href={`https://etherscan.io/address/${data.lpToken}`}
```

Comment:
please use `getMultichainScanLink`

Reply:
Thank you.
I was going to make such function.